src/drivers/auth/index.ts

```diff
@@ -108,6 +98,26 @@ const loadCredentialsWithAutoLogin = async (options?: {
   }
 };
 
+export const remainingTokenTime = (identity: Identity) =>
+  identity.credential.expires_at - Date.now() * 1e-3;
+
+const loadCredentialsWithAutoLogin = async (options?: {
+  noRefresh?: boolean;
+}): Promise<Identity> => {
+  const identity = await loadCredentials();
+  if (remainingTokenTime(identity) > 60) {
```

Comment:
tiniest of nits, but we could save this magic value somewhere so that it's easier to handle in the future and possible also self-document like if these are MS or S.